### DISCUSSION TITLE
Support for ServiceHealthBehavior

### DISCUSSION THREAD

**Discussion by plombaer** [CoreWCF/CoreWCF#1333] [General]
Hello,
We have a Net48 custom implementation of ServiceHealthBehavior. We would like to migrate to Net8/CoreWcf but I don't find an equivalent.
Do you have a solution to achieve the same as ServiceHealthBehavior ?

Thanks for your help

**Comment by g7ed6e** [comment]
Hi @plombaer i think you can achieve the same using ASP.NET Core health checks. Please see https://learn.microsoft.com/en-us/aspnet/core/host-and-deploy/health-checks?view=aspnetcore-8.0

**Reply by plombaer** [reply]
Hi @g7ed6e 
Thanks. I will try that.
